docs/dashboards/synchronize-components.md

```diff
@@ -1,31 +1,29 @@
-Synchronizing components
+Synchronizing `Dashboards` components
 ===
 
-Components may require synchronization that aids in visualizing, navigating
-and highlighting specific data.
+In addition to sharing data via the data pool, `Dahsboards` components can use the synchronization mechanism to aid visualization, navigation and highlighting of specific data.
```

Comment:
Dahsboards -> Dashboards, same thoughts about \`, maybe \*\* will be better.

Reply:
The same as above, let's use `**Dashboard**` instead.